Configuration/Caches.yaml

```diff
@@ -0,0 +1,2 @@
+FlowPackJobQueueCommon_MessageCache:
+  frontend: Neos\Cache\Frontend\VariableFrontend
```

Comment:
I'd suggest to add `persistent: true` here, because this should not be flushed / whatever. It's a rare case that could happen during a deployment or flush, but anyway...